sec_certs_page/templates/index.html.jinja2

```diff
@@ -8,14 +8,16 @@
                     <h1>About sec-certs</h1>
                     <p class="lead fw-light pt-2">
                         We want to become the one-stop shop to explore the Common Criteria / FIPS 140
-                        certification ecosystem. We aggregate and annotate certification data, enabling
-                        unified search, vulnerability investigation, trend analyses, side-by-side
-                        comparisons, notifications and other features.
+                        certification ecosystem. We aggregate and annotate certification data, enabling 
+                        you to perform unified search, vulnerability investigation, trend analyses, side-by-side 
+                        comparisons, notifications and other features for certified products.
```

Comment:
The "you" is not the one doing a lot of these things (e.g. notifications or "other features").